utils/release-candidate-testing/node/package.json

```diff
@@ -0,0 +1,22 @@
+{
+    "name": "release-candidate-testing-nodejs",
+    "version": "1.0.0",
+    "description": "A small amount of test cases to test wether release candidate in working fine",
+    "main": "index.ts",
+    "type": "module",
+    "scripts": {
+        "test": "node index.js",
+        "prettier:check:ci": "./node_modules/.bin/prettier --check .",
+        "prettier:format": "./node_modules/.bin/prettier --write . "
+    },
+    "author": "",
+    "license": "Apache-2.0",
+    "dependencies": {
+        "@aws/glide-for-redis": "^0.3.3",
```

Comment:
don't we wont to pass a specific version - the RC version?

Reply:
We do, we will install the @next version during the CI and it'll replace it.
But i want a way to run it locally in order to test the file, otherwise its shown all read.
I can use latest instead hard version.